practicalai/practical-ai-74.md

```diff
@@ -28,9 +28,9 @@ If you were a long-time listener of the podcast - we've been doing this for abou
 
 **Daniel Whitenack:** That's awesome. You mentioned a few things, like you held different positions like data engineering, and machine learning engineering, and data science, and computational research... Sometimes these sort of titles get a little bit fuzzy. I'm wondering, from your perspective, I guess 1) how you would define what you did differently as a data engineer, versus maybe some of the more sciency things... And then also if that more focused engineering experience influenced how you do machine learning.
 
-**Tania Allard:** Yeah, right. I can say I've been working across the machine learning pipeline in all the different roles... And as you mentioned, a lot of these roles are very \[unintelligible 00:05:29.20\] or have a lot of things in common. When people talk about data scientist, and data engineering roles in machine learning research, or machine learning engineering rather, they try to use these Venn diagrams... And I've found that it is not very descriptive.
+**Tania Allard:** Yeah, right. I can say I've been working across the machine learning pipeline in all the different roles... And as you mentioned, a lot of these roles are very, very close or they have a lot of things in common. When people talk about data scientist, and data engineering roles in machine learning research, or machine learning engineering rather, they try to use these Venn diagrams... And I've found that it is not very descriptive.
 
-For example, if you're working on the data science side of the pipeline, you're focusing much more on the statistics, on developing novel algorithms or models that would help your business or your company to get \[unintelligible 00:06:03.07\] good insights. But then you will probably have/need some software engineering skills as well, to take that into a production format with the rest of your dev environment or your dev team... Whereas when you're working on the data engineering side of things, you're focusing much more on all the processes that are \[unintelligible 00:06:23.24\] but sometimes you still have to know how that data is gonna be integrated in your models, so that the data is actually usable by the rest of the team.
+For example, if you're working on the data science side of the pipeline, you're focusing much more on the statistics, on developing novel algorithms or models that would help your business or your company to get most of their data or get good insights. But then you will probably have/need some software engineering skills as well, to take that into a production format with the rest of your dev environment or your dev team... Whereas when you're working on the data engineering side of things, you're focusing much more on all the processes that are, instructions or transform load, but sometimes you still have to know how that data is gonna be integrated in your models, so that the data is actually usable by the rest of the team.
```

Comment:
Sounds like Tania is saying "[extraction, transform, load](https://en.wikipedia.org/wiki/Extract,_transform,_load)"
```suggestion
For example, if you're working on the data science side of the pipeline, you're focusing much more on the statistics, on developing novel algorithms or models that would help your business or your company to get most of their data or get good insights. But then you will probably have/need some software engineering skills as well, to take that into a production format with the rest of your dev environment or your dev team... Whereas when you're working on the data engineering side of things, you're focusing much more on all the processes that are extraction, transform, load, but sometimes you still have to know how that data is gonna be integrated in your models, so that the data is actually usable by the rest of the team.
```